src/commands/dev/headers.rs

```diff
@@ -1,10 +1,35 @@
 const HEADER_PREFIX: &str = "cf-ew-raw-";
 
+use std::str::FromStr;
+
 use hyper::header::{HeaderMap, HeaderName};
 use hyper::http::request::Parts as RequestParts;
 use hyper::http::response::Parts as ResponseParts;
+use hyper::http::status::StatusCode;
+
+pub fn structure_request(parts: &mut RequestParts) {
```

Comment:
reason i did this is because it matches the function name of the response header function - structure_request and destructure_response - i dont necessarily anticipate putting more stuff into either of these but this feels like a more consistent structure to me. i struggled with figuring out how to name this stuff.

maybe @gabbifish has some ideas

Reply:
i think the `structure_request` name is great. my only comment is, until you need to split out and differentiate steps, there's not much need for the `prepend_request_headers` function. however, it is pretty informative, so i think it's fine either way.

Reply:
This is definitely a readability thing, I figure the compiler will optimize this nested call logic out anyways. I don't exactly think we need to have `destructure_response` and `structure_request` complement each other; this example is distinct from `prepend_request_headers_prefix` and `strip_response_headers_prefix` because they do the opposite of each other (one adds header prefixes, the other removes those exact header prefixes). Tl;dr I think we could get rid of this nested call.